src/controllers/auth.js

```diff
@@ -67,7 +84,64 @@ export const register = async (req, res) => {
       res,
       StatusCodes.INTERNAL_SERVER_ERROR,
       null,
-      error.message
+      error.message,
     );
   }
 };
+
+export const ApproveCompanyActivateUser = async (req, res) => {
+  const { id } = req.params;
+  const { status } = req.body;
+  const user = await findUserById(id);
```

Comment:
Why do we need to specify the user ID in order to activate a company?

I also think this should be in the companies controller